course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -349,8 +362,11 @@ def test_exclude_utm(self):
 class CourseRunWithProgramsSerializerTests(TestCase):
     def setUp(self):
         super().setUp()
+        self.edx_org_short_name = 'edX'
         self.request = make_request()
         self.course_run = CourseRunFactory()
+        # Course_run of edx organization
+        self.course_run2 = CourseRunFactory(course__authoring_organizations__key=self.edx_org_short_name)
```

Comment:
In `setUp` method we declare variables and call methods which are gonna use by multiple test cases. And this is not the case here. Move `edx_org_short_name`, `course_run2` to method `test_filter_course_runs_on_edx_org_short_name`.

Reply:
Done